.github/workflows/haskell-ci.yml

```diff
@@ -8,9 +8,9 @@
 #
 # For more information, see https://github.com/haskell-CI/haskell-ci
 #
-# version: 0.15.20220808
+# version: 0.17.20230827
```

Comment:
FWIW, this file is generated using a fork of `haskell-ci`.

It would be great if @andreasabel made that clear.

Reply:
Yes, I prefer to use `.0` for the RC versions of GHC rather than `.1`.  Using the original `haskell-ci` this would be `tested-with: GHC == 9.8.1` (which I find less true than `tested-with: GHC == 9.8.0`).

Reply:
What I can do about this is to update the text
`For more information, see https://github.com/haskell-CI/haskell-ci` to point to my fork.

Reply:
Ok, did that in https://github.com/andreasabel/haskell-ci/commit/a2949c98d825bf815d1008f43ea02c9200bf7ce3.